src/System.Private.ServiceModel/tests/Scenarios/Client/ChannelLayer/DuplexChannelShapeTests.4.0.0.cs

```diff
@@ -140,30 +140,22 @@ public static void CallbackBehavior_ConcurrencyMode_Single_NetTcpBinding()
         InstanceContext instanceContext;
         DuplexChannelFactory<IWcfDuplexService_CallbackConcurrencyMode> factory = null;
         IWcfDuplexService_CallbackConcurrencyMode channel = null;
-        try
-        {
-            // *** SETUP *** \\
-            binding = new NetTcpBinding(SecurityMode.None);
-            instanceContext = new InstanceContext(new CallbackHandler_ConcurrencyMode_Single());
-            factory = new DuplexChannelFactory<IWcfDuplexService_CallbackConcurrencyMode>(instanceContext, binding, Endpoints.DuplexCallbackConcurrencyMode_Address);
 
-            // *** EXECUTE *** \\
-            channel = factory.CreateChannel();
-            channel.DoWork();
+        // *** SETUP *** \\
+        binding = new NetTcpBinding(SecurityMode.None);
+        instanceContext = new InstanceContext(new CallbackHandler_ConcurrencyMode_Single());
+        factory = new DuplexChannelFactory<IWcfDuplexService_CallbackConcurrencyMode>(instanceContext, binding, Endpoints.DuplexCallbackConcurrencyMode_Address);
 
-            // *** VALIDATE *** \\
-            Assert.True(CallbackHandler_ConcurrencyMode_Single.s_manualResetEvent.WaitOne(20000));
-            Assert.Equal(1, CallbackHandler_ConcurrencyMode_Single.s_counter);
+        // *** EXECUTE *** \\
+        channel = factory.CreateChannel();
+        channel.DoWork();
 
-            // *** CLEANUP *** \\
-            factory.Close();
-            ((ICommunicationObject)channel).Close();
-        }
-        finally
-        {
-            // *** ENSURE CLEANUP *** \\
-            ScenarioTestHelpers.CloseCommunicationObjects((ICommunicationObject)channel, factory);
-        }
+        // *** VALIDATE *** \\
+        Assert.True(CallbackHandler_ConcurrencyMode_Single.s_manualResetEvent.WaitOne(20000));
```

Comment:
Make s_manualResetEvent an instance field or property. When you create an instance of the class CallbackHandler_ConcurrencyMode_Single and pass it to the InstanceContext constructor, keep a reference to it and then you can access the manual reset event via the instance reference rather than statically.